app/views/contexts/calculated_summary_context.py

```diff
@@ -167,6 +171,7 @@ def _get_formatted_total(
         groups: list,
         current_location: Location,
         calculation: Union[Callable, ImmutableDict],
+        routing_path: RoutingPath,
```

Comment:
Same as above.

Reply:
The only thing this method needs should be groups and calculation.

Reply:
Removed these now 👍